### ISSUE TITLE
`.toc.item` doesn't exist?

### ISSUE THREAD

**Issue by ckcollab** [codalab/competitions-v2#74]
![image](https://user-images.githubusercontent.com/2185159/44503109-7759d700-a649-11e8-8379-cafb33817592.png)

This may be due to some kind of race condition, like the elements don't exist yet?

This is probably the culprit, in `init.js`:

```javascript
    // create sidebar and attach to menu open
    $('.ui.sidebar').sidebar('attach events', '.toc.item')
```

**Comment by BrendanMyers88** [comment]
Resolved by PR #75

**Comment by BrendanMyers88** [comment]
Resolved by #75
